conduct.md

```diff
@@ -166,20 +184,19 @@ It is inevitable that conflicts will arise when working as a large group. It is
   * Members shall
     * Resolve conflicts before they become severe by engaging in respectful conversations.
     * If conflicts become severe between class members, they should be brought
-    to the attention of the project managers who will work to resolve any issues.
+      to the attention of the project managers who will work to resolve any issues.
     * Project managers, technical leaders and the course instructor are
-    responsible for resolving any issues by proposing solutions to reach a
-    compromise between class members involved.
+      responsible for resolving any issues by proposing solutions to reach a
+      compromise between class members involved.
     * Project managers may reassign different tasks to group members to solve a conflict.
     * The course instructor has a final say in how severe conflicts should be resolved.
-    * :heavy_exclamation_mark: TODO: add any additional items in the A category for the conflicts.
 
 * Unacceptable Conduct (U):
   * Members shall not
     * Encourage or engage in conflicts that undermine the code of conduct.
     * Escalate existing conflicts that jeopardize the group project.
     * Disregard or refuse to resolve an existing conflict that has been brought to their attention.
-    * :heavy_exclamation_mark: TODO: add any additional items in the U category for the conflicts.
+    * Primarily discuss conflicts and issues with classmates with outside classmates and friends rather than with those they wish to resolve them with.
```

Comment:
I think this needs be reworded. It is a little repetitive with "classmates" and "outside classmates"